pkg/kubelet/events/event.go

```diff
@@ -77,6 +77,11 @@ const (
 	FailedPrepareDynamicResources        = "FailedPrepareDynamicResources"
 	PossibleMemoryBackedVolumesOnDisk    = "PossibleMemoryBackedVolumesOnDisk"
 	CgroupV1                             = "CgroupV1"
+	PodResizeInProgress                  = "PodResizeInProgress"
+	PodResizeDeferred                    = "PodResizeDeferred"
+	PodResizeInfeasible                  = "PodResizeInfeasible"
+	PodResizeFailed                      = "PodResizeFailed"
+	PodResizeCompleted                   = "PodResizeCompleted"
```

Comment:
It might be better to put those reasons into "Pod event reason list" above though it doesn't seem so strict.

Reply:
make sense! thanks for this. done